src/views/studio/studio-image.jsx

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable react/jsx-no-bind */
+import React from 'react';
+import PropTypes from 'prop-types';
+import {connect} from 'react-redux';
+
+import {selectStudioImage, selectIsLoadingInfo} from '../../redux/studio';
+import {selectCanEditInfo} from '../../redux/studio-permissions';
+import {
+    mutateStudioImage, selectIsMutatingImage, selectImageMutationError
+} from '../../redux/studio-mutations';
+import Spinner from '../../components/spinner/spinner.jsx';
+
+const StudioImage = ({
+    imageError, isLoading, isMutating, image, canEditInfo, handleUpdate
+}) => (
+    <div>
+        <h3>Image</h3>
+        {isLoading ? (
```

Comment:
Just because you specifically called out the language around "loading"-- I noticed here and in the other views the use of "loading" where "fetching" might be more precise. Not sure that needs to be changed in this PR, necessarily.